packages/admin/cms-admin/src/pages/createUsePage.tsx

```diff
@@ -449,8 +461,22 @@ function PageSaveButton({ handleSavePage, hasChanges, saving, saveError }: PageS
             ) : undefined,
     };
 
+    const splitButtonProps: Partial<SplitButtonProps> = {};
+
+    if (hasConflict) {
+        saveButtonProps.saveIcon = <ErrorIcon />;
+        saveButtonProps.color = "error";
+        saveButtonProps.sx = {
+            color: (theme) => theme.palette.error.contrastText,
+            [`&.${buttonGroupClasses.grouped}:not(:last-child)`]: {
+                borderRightColor: (theme) => theme.palette.error.contrastText,
```

Comment:
Should the border really be white? 
In all other states the border is slightly darker than the background 🤷🏼 

```suggestion
                borderRightColor: (theme) => theme.palette.error.dark,
```

Reply:
Here we are talking about the line in the middle here: 

<img width="159" alt="Bildschirmfoto 2023-11-13 um 17 04 42" src="https://github.com/vivid-planet/comet/assets/13380047/2bfe3ad4-94a4-43f6-b6b1-08136b9b3eca">

So I don't think it should be dark red. Rather white or off-white. Like F2F2F2 (neutrals 50)

Reply:
I consulted with UX and changed it to dark red

<img width="446" alt="Bildschirmfoto 2023-11-15 um 09 27 39" src="https://github.com/vivid-planet/comet/assets/13380047/5636c3c0-e254-432b-ac6f-e17534cca311">